capstone/capweb/templates/docs/02_site_features/03_api.md

```diff
@@ -133,13 +133,22 @@ Many parameters can be appended with `__in`, `__gt`, `__gte`, `__lt`, or `__lte`
     * __description:__  citation to case, e.g. `1 Ill. 21`
 * `cites_to`
     * __data type:__    string or integer
-    * __description:__  find cases that cite to the given citation or case ID, e.g. `1 Ill. 21` or `12345`
+    * __description:__  find cases that cite to the given citation or case ID, e.g. `1 Ill. 21` or `12345`. 
 * `cites_to.reporter`
     * __data type:__    string
     * __description:__  find cases that cite to the given reporter, e.g. `Ill. 2d` or `Harv. L. Rev.`
 * `cites_to.category`
     * __data type:__    string
-    * __description:__  find cases that cite to the given citation category, e.g. `reporter:state` or `reporters:federal`. Category values can be found in results for the cases endpoint under `extracted_citations[].category`
+    * __description:__  find cases that cite to the given citation category, e.g. `reporter:state` or `reporters:federal`. Category values can be found in results for the cases endpoint under `extracted_citations[].category`.
+* `opinion__cites_to`
```

Comment:
We should chat offline about whether to add this prefix or just say that `author` is ANDed with `cites_to` (and `search`?). I don't quite have it figured out.

Reply:
yeah, that conversation makes sense - i originally had that logic here, but was scared to miss the scenario where users are only curious about citations to a case as some signal for a topic that they want to filter on and then retrieve the opinions of a particular justice, versus the scenario where we want to think about when a justice *makes a cite*. happy to learn a bit more about the problem space